src/app/views/experiment_page.js

```diff
@@ -0,0 +1,68 @@
+import View from 'ampersand-view';
+import dom from 'ampersand-dom';
+import mustache from 'mustache';
+import webChannel from '../lib/web_channel';
+
+export default View.extend({
+  template: mustache.render(`<section class="page" data-hook="experiment-page">
+               <h1 data-hook="displayName"></h1>
+               <button data-hook="install">Install</button>
+             </section>`),
+
+  events: {
+    'click button': 'onClick'
+  },
+
+  initialize(opts) {
+    View.prototype.initialize.apply(this, arguments);
+    if (opts && opts.experiment) {
+      this.model = opts.experiment;
+    }
+  },
+
+  render(opts) {
+    View.prototype.render.apply(this, arguments);
+    this.renderWithTemplate(this);
+
+    if (opts && opts.experiment) {
+      this.model = opts.experiment;
+    }
+    this.heading = this.el.querySelector('[data-hook=displayName]');
+    dom.text(this.heading, this.model.displayName);
+
+    this.button = this.el.querySelector('button');
+    this.renderButton();
+  },
+
+  // TODO: move into a base view?
+  remove() {
+    const parent = this.el.parentNode;
+    if (parent) parent.removeChild(this.el);
+  },
+
+  renderButton() {
+    if (this.model.isInstalled) {
+      dom.setAttribute(this.button, 'data-hook', 'uninstall');
+      dom.text(this.button, 'Uninstall');
+    } else {
+      dom.setAttribute(this.button, 'data-hook', 'install');
+      dom.text(this.button, 'Install');
+    }
+  },
+
+  onClick(evt) {
```

Comment:
You know, this is actually a really interesting and subtle point.

I've always used names like `onClick` or `onKeyDown` to indicate that a given function was a DOM event handler. Similarly, I've used names like `onFoo` to indicate that a function is a handler for a `foo` event.

However, we've got this nice declarative syntax for binding view methods to the DOM. We don't have to guess at what triggers a given view method, since a few lines up, we've got 
`'click [data-hook=foo]': 'install'`. It's nice to not lose the DOM -> binding link, but also be able to name the handlers things that have more application meaning than DOM meaning.